src/Employer/Employer.Web/Orchestrators/Part2/QualificationsOrchestrator.cs

```diff
@@ -136,23 +149,14 @@ private void SyncErrorsAndModel(IList<EntityValidationError> errors, Qualificati
                 error.PropertyName = error.PropertyName.Substring(error.PropertyName.LastIndexOf('.') + 1);
             }
             
-            //Remove other qualification errors
+            //Remove other qualification errors - there shouldn't be any
```

Comment:
uh... what?

Reply:
Ha! Yes that's fair enough. It was an aide memoir for my debugging. I have now removed my addition.